pnpm-lock.yaml

```diff
@@ -205,36 +209,6 @@ importers:
         specifier: ^7.5.4
         version: 7.5.4
 
-  external/zodui/packages/core:
-    dependencies:
-      zod:
-        specifier: ^3.21.4
-        version: 3.21.4
-
-  external/zodui/packages/plugins/common: {}
-
-  external/zodui/packages/react:
-    dependencies:
-      tdesign-react:
-        specifier: ^1.0.3
-        version: 1.0.3(react-dom@18.2.0)(react@18.2.0)
-      zod:
-        specifier: ^3.21.4
-        version: 3.21.4
-    devDependencies:
-      '@types/react':
-        specifier: ^18.0.28
-        version: 18.2.22
-      less:
-        specifier: ^4.1.3
-        version: 4.1.3
-      react:
-        specifier: ^18.2.0
-        version: 18.2.0
-      sass:
-        specifier: ^1.59.3
-        version: 1.67.0
-
```

Comment:
This doesn't look like it should be submitted.

Reply:
I notice `external/zodui` is a part of this project. So i install the submodule and it's work well :)